src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let costs = days * 40;
```

Comment:
From the cheklist: _Don't use literals in expressions (HARDCODE). Instead use constants explaining this values_
Create proper const variables